tests/Microsoft.Bot.Schema.Tests/Teams/MessageActionsPayloadTests.cs

```diff
@@ -0,0 +1,388 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using Microsoft.Bot.Schema.Teams;
+using Xunit;
+
+namespace Microsoft.Bot.Schema.Tests.Teams
+{
+    /// <summary>
+    /// Tests to ensure that MessageActionsPayload works as expected.
+    /// </summary>
+    public class MessageActionsPayloadTests
+    {
+        /// <summary>
+        /// Ensures the constructor of the <see cref="MessageActionsPayload"/> class works as expected.
+        /// </summary>
+        [Fact]
+        public void TestMessageActionsPayloadConstructor()
+        {
+            var messageActionPayload = new MessageActionsPayload();
+
+            Assert.NotNull(messageActionPayload);
+        }
+
+        /// <summary>
+        /// Ensures the constructor of the <see cref="MessageActionsPayload"/> class works as expected with arguments.
+        /// </summary>
+        [Fact]
+        public void TestMessageActionsPayloadConstructorWithArguments()
+        {
+            var messageActionPayload = new MessageActionsPayload
+            {
+                Id = "testId",
+                CreatedDateTime = DateTime.Today.ToString(),
+                Deleted = false,
+                Importance = "normal",
+                Locale = "en-us",
+                From = new MessageActionsPayloadFrom(),
+                Body = new MessageActionsPayloadBody(),
+                Attachments = new List<MessageActionsPayloadAttachment>(),
+                Mentions = new List<MessageActionsPayloadMention>(),
+                LinkToMessage = new Uri("https://teams.microsoft.com/l/message/testing-id")
+            };
+
+            Assert.NotNull(messageActionPayload);
+        }
+
+        /// <summary>
+        /// Ensures that the Id property can be set and retrieved.
+        /// </summary>
+        [Fact]
+        public void TestGetId()
+        {
+            var id = "testId";
+            var activity = new Activity
+            {
+                Type = ActivityTypes.Invoke,
+                Name = "composeExtension/submitAction",
+                ChannelData = new MessagingExtensionAction { MessagePayload = new MessageActionsPayload { Id = id } }
+            };
+
+            var channelData = activity.GetChannelData<MessagingExtensionAction>();
+
+            Assert.Equal(id, channelData.MessagePayload.Id);
+        }
+
+        /// <summary>
+        /// Ensures that the ReplyToId property can be set and retrieved.
+        /// </summary>
+        [Fact]
+        public void TestGetReplyToId()
```

Comment:
This test seems to be testing MessagingExtensionAction, the focus of this test class is to test MessageActionPayload, I would suggest we remove it.